i18n/en-US.properties

```diff
@@ -876,8 +876,12 @@ boxui.datePicker.chooseDate = Choose Date
 boxui.datePicker.dateClearButton = Clear Date
 # Date entered is invalid
 boxui.datePicker.iconAlertText = Invalid Date
+# Message for screenReader users when trying to mention a user but there are no matches
+boxui.draftjs.mentionSelector.noUsersFound = No users found
 # Message to display when a user triggers a mention
 boxui.draftjs.mentionSelector.startMention = Mention someone to notify them
+# Message for screenReader users when a certain number of users is being mentioned
+boxui.draftjs.mentionSelector.usersFound = {usersFound} users found
```

Comment:
Should we account for single vs. plural in this message?

Reply:
I don't think it is required "1 users found", is ok for screenReader users

Reply:
Shouldn't be a hard fix, see https://github.com/box/box-ui-elements/blob/830bf71bef5720a5e6ba128afa35152112674c17/src/elements/content-sidebar/versions/messages.js#L115 for an example